### PR TITLE
Work on Redux: reducers and actions

### PR DESCRIPTION
**Pull request by jaimevillegas**
Hi There!

I have made the following changes to the code:

- Setup Redux Toolkit. :rocket:
- Create a directory tree as described in the activity. :card_file_box:
- Configure the Redux store `store.js`. :wrench:
- Define a slice of state for books `booksSlice.js`. :zap:
- Define a slice of state for categories `categoriesSlice.js`. :zap:

Thank you for taking the time to review my code! :fire:

### CODE REVIEW

File: src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {
+    booklist: 0,
+    categories: 0,
```
Comment (mckent05) [inline src/redux/store.js]: - According to the project requirements, you are expected to use the reducers from the `bookSlice` and `categoriesSlice` in the redux store. Please import them and use them in the store 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @jaimevillegas, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job adding reducers and actions to your application, welldone 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/store.js (hunk 1 above)
Comment (jaimevillegas) [reply src/redux/store.js]: Hello @mckent05!

Thank you for the comment! I have solved it now!

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @jaimevillegas ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitFlow :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**